ikabot/function/constructionList.py

```diff
@@ -10,33 +10,42 @@
 import traceback
 import threading
 from decimal import *
+from typing import List
```

Comment:
probably not needed

Reply:
yep, almost definitely, may be we should just remove to keep imports at a minimum